test/test_operations_fallback.py

```diff
@@ -0,0 +1,47 @@
+# Parse local options first, and rewrite the sys.argv[].
+# We need to do that before import "common", as otherwise we get an error for
+# unrecognized arguments.
+import argparse
+import sys
+
+parser = argparse.ArgumentParser(add_help=False)
+parser.add_argument('--replicated', action='store_true')
+parser.add_argument('--long_test', action='store_true')
+parser.add_argument('--max_diff_count', type=int, default=25)
+parser.add_argument('--verbosity', type=int, default=0)
+FLAGS, leftovers = parser.parse_known_args()
+sys.argv = [sys.argv[0]] + leftovers
+
+# Normal imports section starts here.
+import torch
+import torch_xla
+import torch_xla.utils.utils as xu
+import torch_xla.core.xla_model as xm
+import torch_xla.debug.metrics as met
+import unittest
+
+
+class TestOperationsFallback(unittest.TestCase):
```

Comment:
hmm, is fallback a correct name? Fallback means a ops fallback to pytorch native CPU. I think the purpose of this test is more of we want to check we still get the whole op instead some decompositied ops. Maybe `TestOperationsHLO`?

Reply:
Sure, I'll rename `fallback` to `hlo`.